tests/bits/find_cell_5.output

```diff
@@ -1,3 +1,3 @@
 
-DEAL::3.462
-DEAL::<0.750000 0.625000 0.625000> <0.875000 0.625000 0.625000> <0.750000 0.750000 0.625000> <0.875000 0.750000 0.625000> <0.750000 0.625000 0.750000> <0.875000 0.625000 0.750000> <0.750000 0.750000 0.750000> <0.875000 0.750000 0.750000> 
+DEAL::3.455
+DEAL::<0.625000 0.625000 0.625000> <0.750000 0.625000 0.625000> <0.625000 0.750000 0.625000> <0.750000 0.750000 0.625000> <0.625000 0.625000 0.750000> <0.750000 0.625000 0.750000> <0.625000 0.750000 0.750000> <0.750000 0.750000 0.750000> 
```

Comment:
Yes, the point is on the interface between cells. (I think in a corner where 8 cells meet.) Since the new formula reorders the floating point operations a bit in terms of the tensor inversion, the initial guess changes in the roundoff region and we pick something else. The three affected tests belong to the category where a more robust approach that allows two, four or eight different answers depending on whether the point belongs to a face, edge or vertex.